k8s-manifest/action.yaml

```diff
@@ -26,27 +22,16 @@ inputs:
       Sets the score threshold for scans. Scores that fall below the threshold will exit 1. (default "0" - job continues regardless of the score returned by a scan).
     default: "0"
     required: false
-  service-account-credentials:
-    description: "Base64 encoded service account credentials used to authenticate with Mondoo Platform"
-    required: true
 runs:
-  using: "composite"
-  steps:
-    - name: Install Mondoo Client
-      shell: bash
-      run: |
-        echo Installing Mondoo Client...
-        echo ${{ inputs.service-account-credentials }} | base64 -d > mondoo.json
-        curl -sSL https://mondoo.com/install.sh | bash
-    - name: Mondoo status
-      shell: bash
-      run: mondoo status --config mondoo.json
-    - name: Run mondoo scan ${{ inputs.asset-type }}
-      shell: bash
-      run: >
-        mondoo scan k8s ${{ inputs.path }}
-        --log-level ${{ inputs.log-level }}
-        --output ${{ inputs.output }}
-        --score-threshold ${{ inputs.score-threshold }}
-        ${{ inputs.args }}
-        --config mondoo.json
+  using: "docker"
+  image: "docker://mondoo/cnspec"
```

Comment:
won't this always pull the latest version? Is this what we want?

Reply:
because this is running in a container, don't you need to mount some folder? I don't know how actions work in details but if the checkout of the repo is done on the runner and we have cnspec in a container, I guess cnspec won't be able to read the checkout by default?

Reply:
About the version: We currently do not have a v7 tag or something similar. So, I guess `latest` is good for now. I'll open an issue to create a v7 tag.

About the path: GitHub does some magic when starting the action and mounts a lot of directories, e.g.: https://github.com/mondoohq/actions/actions/runs/3427042238/jobs/5709557563

Reply:
Makes sense! Please open an issue for major version pinning since I personally had the negative experience of my actions breaking 3 times right after I fixed them because of different breaking changes we did 😄

Reply:
https://github.com/mondoohq/cnspec/issues/174

Reply:
I like to add a major version tag. For github actions I would prefer if we always use latest. Since we have a non-breaking commitment since needs to work. If not we need to increase testing in cnquery / cnspec to ensure it is not breaking. Therefore I would make version pinning optional.

Reply:
Similar to our operator we should be pinning to the major so we can do our own post release validation. That way we don't accidentally break all our customer pipelines when we release v8.